superflore/utils.py

```diff
@@ -47,7 +47,7 @@ def get_pr_text(comment=None, markup='```'):
     msg = ''
     if comment:
         msg += '%s\n' % comment
-    msg += 'To reproduce this PR, run the following command.\n\n'
+    msg += 'This pull request was generated by the following command:\n\n'
```

Comment:
Sorry to be pedantic, but I'd really prefer `running the following command:`

Reply:
No worries; done!